src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties

```diff
@@ -19,59 +19,38 @@
 
 # default org.apache.maven.plugins groupId
 maven-archetype-plugin=3.2.0
-# https://issues.apache.org/jira/browse/ARCHETYPE-590
 maven-assembly-plugin=3.2.0
-# https://issues.apache.org/jira/browse/MASSEMBLY-921
 maven-jar-plugin=3.2.0
-# https://issues.apache.org/jira/browse/MJAR-263
-# https://issues.apache.org/jira/browse/MJAR-275
 maven-ejb-plugin=3.1.0
-# https://issues.apache.org/jira/browse/MEJB-128
 maven-plugin-plugin=3.5.1
-# https://issues.apache.org/jira/browse/MPLUGIN-326
 maven-remote-resources-plugin=1.7.0
-# https://issues.apache.org/jira/browse/MRRESOURCES-114
-maven-shade-plugin=3.2.3
-# https://issues.apache.org/jira/browse/MSHADE-352
+maven-shade-plugin=fail:https://issues.apache.org/jira/browse/MSHADE-420
 maven-source-plugin=3.2.1
-# https://issues.apache.org/jira/browse/MSOURCES-123
 maven-war-plugin=3.3.1
-# https://issues.apache.org/jira/browse/MWAR-432
 maven-ear-plugin=3.1.0
-# https://issues.apache.org/jira/browse/MEAR-280
 maven-rar-plugin=3.0.0
-# https://issues.apache.org/jira/browse/MRAR-86
 maven-acr-plugin=3.2.0
-# https://issues.apache.org/jira/browse/MACR-53
 
 # plugin-specific groupId
 
 org.antlr+antlr3-maven-plugin=fail:https://github.com/antlr/antlr3/pull/195
 
-org.apache.felix+maven-bundle-plugin=5.1.4
-# https://issues.apache.org/jira/browse/FELIX-6495
-# https://issues.apache.org/jira/browse/FELIX-6496
-org.apache.karaf.tooling+karaf-maven-plugin=fail:https://issues.apache.org/jira/browse/KARAF-7367
+org.apache.felix+maven-bundle-plugin=5.1.5
+org.apache.karaf.tooling+karaf-maven-plugin=4.3.7
 org.apache.nifi+nifi-nar-maven-plugin=1.3.4
-# https://issues.apache.org/jira/browse/NIFI-9857
-org.apache.servicemix.tooling+depends-maven-plugin=fail:https://issues.apache.org/jira/browse/SM-5021
+org.apache.servicemix.tooling+depends-maven-plugin=1.5.0
```

Comment:
issue has been fixed in SCM, but release not done: it's better to show this as unsolved yet

Reply:
You are right, I will revert.